modules/openapi-generator/src/main/resources/rust/model.mustache

```diff
@@ -6,11 +6,39 @@ use serde::{Deserialize, Serialize};
 {{^isEnum}}{{#vendorExtensions.x-rust-has-byte-array}}
 use serde_with::serde_as;
 {{/vendorExtensions.x-rust-has-byte-array}}{{/isEnum}}
+{{#isEnum}}{{#isInteger}}use serde_repr::{Serialize_repr,Deserialize_repr};{{/isInteger}}{{/isEnum}}
```

Comment:
what about the following to avoid the empty line break?

```
{{#isEnum}}
{{#isInteger}}
use serde_repr::{Serialize_repr,Deserialize_repr};
{{/isInteger}}
{{/isEnum}}
```